geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeRegionDUnitTest.java

```diff
@@ -171,6 +178,75 @@ public void describeRegionWithAsyncEventQueue() throws Exception {
         .containsOutput("async-event-queue-id", "queue1");
   }
 
+  @Test
+  public void testDescribeRegionReturnsDescriptionFromAllMembers() throws Exception {
+    CommandStringBuilder csb = new CommandStringBuilder(DESCRIBE_REGION);
+    csb.addOption(DESCRIBE_REGION__NAME, HOSTING_AND_ACCESSOR_REGION_NAME);
+
+    String command = csb.toString();
+
+    CommandResult commandResult =
+        gfsh.executeAndAssertThat(command).statusIsSuccess().getCommandResult();
+
+    GfJsonObject hostingMembersRegionDesc = getMembersRegionDesc(commandResult, "Hosting Members");
+    GfJsonObject hostingMembersTableContent = hostingMembersRegionDesc
+        .getJSONObject("__sections__-0").getJSONObject("__tables__-0").getJSONObject("content");
+
+    GfJsonObject accessorsRegionDesc = getMembersRegionDesc(commandResult, "Accessor Members");
+    GfJsonObject accessorsTableContent = accessorsRegionDesc.getJSONObject("__sections__-0")
+        .getJSONObject("__tables__-0").getJSONObject("content");
+
+    assertThat(hostingMembersRegionDesc.get("Name").toString())
+        .contains(HOSTING_AND_ACCESSOR_REGION_NAME);
+    assertThat(hostingMembersRegionDesc.get("Data Policy").toString()).contains("partition");
+    assertThat(hostingMembersRegionDesc.get("Hosting Members").toString()).contains("server-1",
+        "server-2", "server-3", "server-4");
+    assertThat(hostingMembersTableContent.get("Type").toString()).contains("Region");
+    assertThat(hostingMembersTableContent.get("Name").toString()).contains("data-policy", "size");
+    assertThat(hostingMembersTableContent.get("Value").toString()).contains("PARTITION", "0");
+
+    assertThat(accessorsRegionDesc.get("Name").toString())
+        .contains(HOSTING_AND_ACCESSOR_REGION_NAME);
+    assertThat(accessorsRegionDesc.get("Data Policy").toString()).contains("partition");
+    assertThat(accessorsRegionDesc.get("Accessor Members").toString()).contains("server-5");
+    assertThat(accessorsTableContent.get("Type").toString()).contains("Region", "Partition");
+    assertThat(accessorsTableContent.get("Name").toString()).contains("data-policy", "size",
+        "local-max-memory");
+    assertThat(accessorsTableContent.get("Value").toString()).contains("PARTITION", "0", "0");
+  }
+
+  private static void createHostingAndAccessorRegion() {
+    server1.invoke(() -> {
```

Comment:
you can use VMProvider.invokeInEveryVM call to consolidate these calls into one.

Reply:
Done!